package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "katello",
+  "version": "1.0.0",
+  "scripts": {
+    "test": "jest webpack",
+    "test:watch": "jest webpack --watchAll",
+    "test:current": "jest webpack --watch",
+    "format":
+      "prettier --single-quote --trailing-comma es5 --write 'webpack/**/*.js'",
+    "build": "npm run format && npm run lint",
+    "lint": "eslint --fix webpack/index.js"
```

Comment:
It should imho be `eslint --fix webpack/index.js || exit 0` otherwise npm will print ugly error when there are some lint issues:
```
npm ERR! Linux 3.10.0-514.10.2.el7.x86_64
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "run" "lint"
npm ERR! node v6.11.3
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! katello@1.0.0 lint: `eslint --fix webpack/index.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the katello@1.0.0 lint script 'eslint --fix webpack/index.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the katello package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     eslint --fix webpack/index.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs katello
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls katello
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/vagrant/foreman/katello/npm-debug.log
```

Reply:
BTW it currently fails when you execute `npm run lint` from the foreman due to foreman not knowing about "eslint-plugin-jest". I'm fixing it in:
https://github.com/theforeman/foreman/pull/4946